src/site/blocks/alert.drupal.liquid

```diff
@@ -34,7 +34,9 @@
     {% if expander.fieldTextExpander == empty %}
         {{ expander.fieldWysiwyg.processed }}
     {% endif %}
-
+    {% comment %} <div id="react-applicationStatus" class="static-page-widget" data-widget-type="Form-2346">
```

Comment:
Could we remove this block?

Reply:
@not yet. This is what content will need for the `Save In Progress` widget.